pages/en/contrib/code-contrib.md

```diff
@@ -8,96 +8,5 @@ sidebar: contrib_sidebar
- 1. Make your change. Remember to update tests as well as code! Always
-    run all the tests to assure nothing else was accidentally broken. For bugs, adding a failing test and submitting a pull request usually leads to the bug being fixed quickly. For features, include tests that cover the entire feature. For remote / client facing features, include integration tests in the [loopback-example-app](](https://github.com/strongloop/loopback-example-app).
- 1. Check for unnecessary whitespace with `git diff --check` before committing.
- 1. Make commits of logical units and push them to Github.
- 1. Use a descriptive commit message, and follow
-    [50/72 format](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
- 1. Use GitHub's pull requests to submit the patch. Feel free to issue the pull
-    request as soon as you have something partial to show and get reviewed.
-    You can add more commits to your pull request as you are progressing
-    on the implementation.  The title of the pull request should start
-    with "WIP" for "work in progress" when the pull request is not complete
-    yet.
- 1. Request a code review. Add a specific committer to speed up this process. (for example, `@bajtos`, `@raymondfeng`).
- 1. Make any requested changes and push to your fork. Make sure your changes are still based on the latest code (use `git rebase upstream/master`).
-
- ### How to rebase your branch ###
-
-Our code base is always getting updated and being able to successfully rebase your PR is very important. Here are some simple steps to rebase your code:
-- Checkout your branch which you are submitting the pull request against `git checkout <my_branch>`
-- Add an upstream remote from your fork `git remote add upstream <upstream_repo>`
-- Fetch the latest from the **_upstream_** remote `git fetch upstream`
-- Rebase your code using the interactive mode `git rebase -i upstream/master`
-- If you are happy with the changes and there are no merge conflicts, push the code to your remote branch `git push origin <my_branch>`
-
-## Running tests
-
-All LoopBack projects follow the same convention for running tests. To run a projects test suite, change to the project's directory and run:
-
-```sh
-npm test
-```
-
-## Create example repositories
-
-If you create an example repository for any usage demo, the name of repository should follow the following convention:
-
-`loopback{major_version}-example-{example_name}`
-
-Usually the `{major_version}` is the lastest major version under development.
-If the example aims at a specific version of LoopBack, then use that number as the `{major_version}`.
-The `{example_name}` is a few words which briefly describes the functionality of the example,
-and joined with `-`.
-
-For example, we build a quick start example app for LoopBack 4 and name it as
-[`loopback4-example-hello-world`](https://github.com/strongloop/loopback-next-hello-world).
```

Comment:
`loopback-next-hello-world` has been deprecated and moved to an archive. Let's use https://github.com/strongloop/loopback4-example-shopping instead.